### ISSUE TITLE
SQL Serverless example error

### ISSUE THREAD

**Issue by kszicsillag** [Azure/azure-cli#11149]
Executing the following example command from the docs page
```
az sql db create -g mygroup -s myserver -n mydb --compute-model Serverless --auto-pause-delay 120
```
results the following error
> When creating a severless database, please pass in edition, family, and capacity parameters through -e -f -c

CLI version: 2.0.76

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c1bdd674-b454-e863-c943-9c979b466a92
* Version Independent ID: b15341f9-3f1b-00e0-a1c2-970f25a6ab03
* Content: [az sql db](https://docs.microsoft.com/en-us/cli/azure/sql/db?view=azure-cli-latest)
* Content Source: [src/azure-cli/azure/cli/command_modules/sql/_help.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/sql/_help.py)
* Service: **sql-database**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by Juliehzl** [comment]
cc @weizeng-msft

**Comment by weizeng-msft** [comment]
@kszicsillag We currently do not support auto default service objective level input param population.
But the work is currently in the plan.
I am addressing this issue by updating the help text with exactly what is asked in the error message.

**Comment by kszicsillag** [comment]
@weizeng-msft thanks, my initial thought was that this is more of a documentation issue than a product issue.

**Comment by jaredmoo** [comment]
Fixed by #11199
